src/util.c

```diff
@@ -191,7 +185,7 @@ void print_test_results(struct ntttcp_test_endpoint *tep)
 	PRINT_INFO_FREE(log);
 	/* only show RetransSegs for TCP traffic */
 	if (tepr->endpoint->test->protocol == TCP) {
-		ASPRINTF(&log, "\t retrans segs\t:%"PRIu64, tepr->packets_retransmitted);
```

Comment:
Please test this and make sure it is working and no function (UI text) change here

Reply:
done